### PR TITLE
fix(avatar): Return the conversation data and fix the documentation

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Return the full conversation object instead of (undocumented) custom data
- [x] Fix documentation missing headers

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Review by marcoambrosini — APPROVED** [review]
🙏

File: docs/conversation.md [hunk 1]
```diff
@@ -453,6 +458,11 @@ Get all (for moderators and in case of "free selection") or the assigned breakou
...
         + `404 Not Found` When the conversation could not be found for the participant
 
+	- Data: See array definition in `Get user´s conversations`
+
+## Get conversations avatar (binary)
+
+* ⚠️ Preview - Might be modified before the capability is added
 * Required capability: `avatar`
 * Method: `GET`
 * Endpoint: `/room/{token}/avatar`
```
Comment (ShGKme) [inline docs/conversation.md:468]: Shouldn't we mention `?v=<avatarVersion>` query param here for HTTP Cache?

Comment (nickvergessen) [reply docs/conversation.md:468]: Query parameters are not part of the API, in fact the API does not eve handle the parameter, so you can not get an older version by specifying an older parameter or something a like, so I'd like not to add it to the docs

**Review by vitormattos — APPROVED** [review]

File: docs/conversation.md (hunk 1 above)
Comment (ShGKme) [reply docs/conversation.md:468]: It is not a part of API, but developers should use it. And I'm sure somebody will miss it.

Maybe we can add a note?

In the previous version, an example of the get image UTL (in POST avatar) contained `?v=<avatarVersion>`.

**Pull request merged**